common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3271,6 +3272,15 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(3)
 
+  val WORKER_FLUSHER_LOCAL_GATHER_API_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.worker.flusher.local.gatherAPI.enabled")
+      .internal
+      .categories("worker")
+      .version("0.5.5")
+      .doc("Worker will use gather API if this config is true.")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
Can we have it disabled by default (only for 0.5) since branch-0.5 has been delivered for a relatively long time?

Reply:
done